mne/tests/test_source_estimate.py

```diff
@@ -886,6 +889,39 @@ def test_vec_stc():
     assert_array_equal(normal.data[:, 0], [1, 2, 0, np.sqrt(3)])
 
 
+def test_epochs_vector_inverse():
+    """Test vector inverse consistency between evoked and epochs"""
+
+    raw = read_raw_fif(fname_raw)
+    events = find_events(raw, stim_channel='STI 014')
+    event_id = dict(aud_l=1)  # event trigger and conditions
+    tmin = -0.2  # start of each epoch (200ms before the trigger)
+    tmax = 0.5  # end of each epoch (500ms after the trigger)
+    raw.info['bads'] = ['MEG 2443', 'EEG 053']
+    baseline = (None, 0)  # means from the first instant to t = 0
+    reject = dict(grad=2000e-13, mag=4e-12, eog=150e-6)
+
+    epochs = Epochs(raw, events, event_id, tmin, tmax, baseline=baseline,
+                    reject=reject, preload=True).crop(0, 0.01)
```

Comment:
let's add:
```
assert len(epochs) == 2
```
to make sure we actually got more than one epoch

Reply:
... and do `tmin=0., tmax=0.01, baseline=None` above instead of cropping here (the DC offset shouldn't matter anyway)